src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -29,19 +29,29 @@ internal static class OpsMetadataRuleConverter
             { "Length", new string[] { "string-length-invalid" } },
         };
 
+#pragma warning disable MEN003 // Method is too long
         public static string GenerateJsonSchema(string rulesContent, string allowlistsContent)
+#pragma warning restore MEN003 // Method is too long
         {
             Log.Write(rulesContent);
             Log.Write(allowlistsContent);
 
-            var rules = JsonConvert.DeserializeObject<Rules>(rulesContent);
+            Rules? rules = null;
+            Taxonomies taxonomies = new();
+            try
+            {
+                rules = JsonConvert.DeserializeObject<Rules>(rulesContent);
+                taxonomies = JsonConvert.DeserializeObject<Taxonomies>(allowlistsContent) ?? new();
+            }
+            catch (Exception ex)
+            {
+                Log.Write(ex);
```

Comment:
should add `_errors.Add(Errors.System.ValidationIncomplete());`?

Reply:
done.